src/tests/modify.test.js

```diff
@@ -257,6 +257,21 @@ describe('modify() - basic mutations', () => {
     });
   });
 
+  it('unexistent fields gets ignored', () => {
```

Comment:
Note on why I decided to write this explicit test:

In Remote we have a lot of global customization that run equally across multiple different JSON Schemas.
But we only want to apply the customizations relative to existing fields. Otherwise we'd inject (create) new fields even though they don't exist. That's why we have the new method on this PR, specific to create new fields.